### DISCUSSION TITLE
https://github.com/BassT23/Proxmox

### DISCUSSION THREAD

**Discussion by Gauvino** [tteck/Proxmox#1003] [Ideas]
So, with a friend whe make a script in bash for updating all-lxcs with multiple mode and super feature and we wanted to know if we can integrate it into your helper script.

**Comment by tteck** [comment]
I believe that referring to your Github page would be the most effective solution for this.

**Reply by Gauvino** [reply]
if you want you can just copy it and put credit of the source in the script that all

**Reply by Gauvino** [reply]
the objective is just to make proxmox easier for everyone

**Reply by Gauvino** [reply]
> I believe that referring to your Github page would be the most effective solution for this.

???
